File: cadwiki-nuget/cadwiki.AutoCAD2021.Base.Utilities/Shared/Lines.vb
Imports Autodesk.AutoCAD.DatabaseServices
Imports Autodesk.AutoCAD.Geometry

Public Class Lines
    Public Shared Function TruncateEndpointsToTwoDecimalPlaces(line As Line) As Line
        Dim truncatedStart As Point3d = Points.ToTwoDecimalPlaces(line.StartPoint)
        Dim truncatedEnd As Point3d = Points.ToTwoDecimalPlaces(line.EndPoint)
        Dim truncatedLine As Line = New Line(truncatedStart, truncatedEnd)
        Return truncatedLine
    End Function
End Class

File: cadwiki-nuget/cadwiki.AutoCAD2021.Base.Utilities/Shared/Lines.cs
using Autodesk.AutoCAD.DatabaseServices;

namespace cadwiki.AutoCAD2021.Base.Utilities
{

    public class Lines
    {
        public static Line TruncateEndpointsToTwoDecimalPlaces(Line line)
        {
            var truncatedStart = Points.ToTwoDecimalPlaces(line.StartPoint);
            var truncatedEnd = Points.ToTwoDecimalPlaces(line.EndPoint);
            var truncatedLine = new Line(truncatedStart, truncatedEnd);
            return truncatedLine;
        }
    }
}